docs/src/piccolo/schema/advanced.rst

```diff
@@ -231,3 +231,58 @@ is not present, it will be reflected and returned.
 .. hint:: Reflection will automatically create ``Table`` classes for referenced
     tables too. For example, if ``Table1`` references ``Table2``, then
     ``Table2`` will automatically be added to ``TableStorage``.
+
+-------------------------------------------------------------------------------
+
+How to create custom column types
+------
```

Comment:
Seems minor, but some Restructured Text linters complain if the underline isn't the same length as the title:

```
How to create custom column types
---------------------------------
```

Reply:
Oh sorry didn't realize that. Will fix it now.